test/src/imgur/getTags.test.ts

```diff
@@ -1,8 +1,13 @@
+import { ImgurClient } from 'imgur'
+
 const imgurGetTagsMethod = 'biketag.images.getTags'
+
 describe(imgurGetTagsMethod, () => {
   test.todo(`${imgurGetTagsMethod} method requires ImgurHash from payload`)
 
+  // Validate `data` field for all three branches of payload conditional
   test.todo(`${imgurGetTagsMethod} method resolves data of type Tag[]`)
```

Comment:
💡 Suggestion: Split into three tests to check each branch in conditional
❓ Needed: Can you get me three valid payload examples?

Reply:
@nonara Yes. I can work on the fixtures for these tests if you like. Alternatively, you should be able to run the code and see the results of each method by running the examples/node.js file.

Reply:
Either is fine. With the time constraint, I'm trying to get as much done as possible without having to clock a lot of time doing up front studying. Toward that end, leaning on your experience a bit can help a lot. It's great if you can point me to something to run that will help get the answer.

In `examples/imgur` I see an example with `hash` passed, which covers one state. The three branches to the conditional above represent three states the input `payload` can be in. I need an example of each state to make sure I can test it.